tests/virt_autotest/libvirt_isolated_virtual_network.pm

```diff
@@ -26,11 +26,15 @@ sub run_test {
     my $vnet_isolated_cfg_name = "vnet_isolated.xml";
     virt_autotest::virtual_network_utils::download_network_cfg($vnet_isolated_cfg_name);
 
+    #Stop named.service, refer to poo#175287
+    systemctl("stop named.service") if (is_sle('15+');
```

Comment:
Lack a ')' at the end, which leads to syntax check error.

Reply:
Great! fix up CI test now.